README.md

```diff
@@ -33,170 +28,167 @@ The picture below shows an example of image generated in this way:
 
 ![Example of shape image](./images/shape_00000.png)
 
-# Dependencies
-
-This project needs to load the following Python dependencies:
-
-+ cv2
-+ logging
-+ matplotlib
-+ numpy
-+ pandas
-+ PIL
-+ tensorflow
-
-These dependencies are stored in `requirements.txt` located at the project root. As a remark, the
-code has been run with Python3 (version 3.5).
-
 # Content
 
-The project contains some Python materials designed to illustrate the Tensorflow library (snippets
-and notebooks)
+The project contains some Python materials designed to illustrate the Tensorflow and Keras
+libraries (snippets and notebooks)
 
 + [article](./article) contains the original text of articles that have been published
   on [Oslandia blog](http://oslandia.com/en/blog/) on this topic
++ [deeposlandia](./deeposlandia) contains main Python modules to train and test convolutional
+  neural networks
 + [images](./images) contains some example images to illustrate the Mapillary dataset as well as
   some preprocessing analysis results
 + [notebooks](./notebooks) contains some Jupyter notebooks that aim at describing data or basic
   neural network construction
-+ [sources](./sources) contains Python modules that train a convolutional neural network based on
-  the Mapillary street image dataset
++ [tests](./tests) contains some test modules to guarantee the functioning of a bunch of snippets;
+  it uses the `pytest` framework.
 
-Additionally, running the code may generate extra repositories:
+Additionally, running the code may generate extra subdirectories in the data repository.
 
-+ [checkpoints](./checkpoints) refers to trained model back-ups, they are
-  organized with respect to models
-+ [graphs](./graphs) is organized like `checkpoints` repository, it contains
-  `Tensorflow` graphs corresponding to each neural network
-+ [chronos](./chronos) allows to store some training execution times, if wanted
+# Installation
 
-These repository are located at the data repository root.
+## Requirements
 
-# Running the code
+This project needs to load the following Python dependencies:
 
-This project supposes that you have downloaded the Mapillary image dataset. The
-following program calls are supposed to be made from the `source` repository.
++ cv2
++ keras
++ h5py
++ logging
++ matplotlib
++ numpy
++ pandas
++ PIL
++ tensorflow
```

Comment:
maybe mention the setup.py file with the dependencies and the requitements dev file if needed.